meet_balls_app/templates/meet_balls_app/register.html

```diff
@@ -0,0 +1,53 @@
+{% extends 'base.html' %}
+
+{% block content %}
+<div class="container">
+    <div class="row justify-content-center">
+        <div class="col-md-6 col-lg-4">
+            <h2 class="text-center mb-4">Register</h2>
+            <form method="post">
+                {% csrf_token %}
+                <div class="mb-3">
```

Comment:
Please change these classes names to meaningful names,
It will help us to work with it in CSS files

Reply:
Assuming these names are bootstrap names. Is it possible to change them? Isn't it ruin the current design?

Reply:
Good point. Use it like that for not have some bootstrap issues